mage_ai/server/websocket_server.py

```diff
@@ -540,25 +544,33 @@ async def check_for_messages():
             task = asyncio.create_task(check_for_messages())
             set_current_message_task(task)
 
-    def __format_error(error: List[str]) -> List[str]:
+    @classmethod
+    def format_error(cls, error: List[str], block_uuid: str = None) -> List[str]:
```

Comment:
Can you add the comment to this method? It's a bit hard to understand the logic here.